src/components/Message/hooks/useActionHandlerMML.js

```diff
@@ -0,0 +1,36 @@
+// @ts-check
```

Comment:
this is basically the same as `useActionHandler`
https://github.com/GetStream/stream-chat-react/pull/562/files#diff-aa50df8b4b857ab266247a60ceb2e55f